client/js/shout.js

```diff
@@ -655,7 +655,7 @@ $(function() {
 				favico.badge("!");
 				if (settings.badge && Notification.permission === "granted") {
 					var notify = new Notification(msg.from + " says:", {
-						body: msg.text.trim(),
+						body: msg.text.trim().replace(/\x02|\x1D|\x1F|\x16|\x0F|\x03(?:\d{1,2}(?:,\d{1,2})?)?/, ""),
```

Comment:
While you're at it, change `\d` to `[0-9]`. See http://stackoverflow.com/a/6479605/2200891